### DISCUSSION TITLE
PatchTool 7.05 10/5 更新漢化、字體、修正bug

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#476] [Announcements]
更新7.0中國服漢化。
更新高畫質字體包。
**修正v3.3.1.0中跨界傳送會導致遊戲崩潰的問題。**

以下兩種方法擇一使用即可；請從release下方連結或內文連結下載檔案。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20241005-fixed.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20241005-fixed.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-20241005.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.3.1.0/font-custom-20241005.rar)：高畫質字體包，請取代原本的resource/font資料夾。請參閱[詳細使用說明](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。
- rawexd-zhcn-20241005-fixed.rar：簡體中文CSV，請取代原本resource/rawexd資料夾。
- [backup-20241005.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.3.1.0/backup-20241005.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.3.1.1'>PatchTool 7.05 10/5 更新漢化、字體、修正bug</a>.</em>

**Comment by Liteya** [comment]
自己是沒有撞到跨界跳錯 應該是addon 的檔案吧 我沒有全換
以下是回報


 

量詞問題
`65,TEXT_KINGZF204_05163_SHALOACI_000_017,(-夏勞奇-)<hex:022903EB0203>你沒事吧？<hex:02100103>出現了多少只毒菇怪？你有沒有受傷？`
(隻)

錯誤用法
`104,TEXT_KINGZF203_05162_SHALOACI_000_061,我之前一直以為是夢境。<hex:02100103>當時也是和某個大人一起線上路上行走，<hex:02100103>好像還聽對方聊了車站啊線路前方的土地之類的話題。`
(線路上行走)

`4,TEXT_FESHLX001_05184_SEQ_04,安撫了菲爾考克斯後與他進行了交談。據他所說，他雖然不能使用變化的魔咒，但很擅長髮現負面感情。和米·凱特露天劇場的南瓜頭女人談談。`
(發現)
`95,TEXT_KINGYX101_05013_TELLER05013_000_061,很久以前，遠征北部大陸的尤卡巨人族中，<hex:02100103>有一個派繫在某個山谷的深處發現了一座遠古時期的建築遺跡。`
(派系)
`51,TEXT_KINGMF113_04958_CLEYRACURATOR_000_011,沒錯，就是這裡！<hex:02100103>它們是我的一點陣圖萊尤拉的朋友訓練的，<hex:02100103>是不是很可愛啊！`
(一位)
(遠程控制 遠程式控制製?) 這樣很繞口..
`133,TEXT_KINGMD102_04913_TONAWAWTABANDIT_000_112,該死！那個小兔崽子！<hex:02100103>等著吧，到時候我要把你的耳朵撕爛，<hex:02100103>再把那個老太太掛起來喂狗……`
(餵狗)
`99,TEXT_KINGMC110_04906_BAKOOLJAJA_000_076,(-巴庫加加勇戰之頭-)因為我認可了烏克·拉瑪特，<hex:02100103>這點陣圖萊尤拉的公主的確值得信任。`
(這位)
`89,TEXT_KINGLA307_04988_HALNARUK_000_119,不是` 才沒有！<hex:02100103>才不是 朋友呢！<hex:02100103>莫布一直 冷麵無私 工作！
(冷面)
`58,TEXT_KINGMB108_04887_ERENVILLE_000_011,走吧，我們去把海牛抓回來。<hex:02100103>這條河的下游有一群性情兇猛的野生海牛，<hex:02100103>逃走的那只要避開它們只能向反方向跑……我們去上游找。`
(那隻)
`61,TEXT_KINGMA107_04866_WUKLAMAT_000_029,你就是選王之人……<hex:02100103>老爹讓重游他的旅途，聽他的果然沒錯！`
(遊)
`85,TEXT_KINGBA511_04843_TEPEKE_000_310,果然，施法需要進行特定的程式吧，<hex:02100103>可那時他什麼也沒做，<hex:02100103>怎麼就啟動了祭器呢？`
(程序 手法, 程式太怪)
`54,TEXT_KINGBA501_04842_TSUUHE_000_006,就這樣，鬥轉星移……村子再也沒有遭遇危險，<hex:02100103>知曉祭器力量的人也都在多年前去世了。`
(斗轉星移)
(御魔 禦魔 ? 其實有點怪)
----------------
建議改英文
`25,TEXT_KINGZD106_05105_TODO_01,在「說話」頻道下輸入「seek the sun」<hex:02100103>向霍比格族男性傳話`
`52,TEXT_KINGZD106_05105_QUIETBOONEWA05105_000_004,要是再失去了丈夫，我真不知該怎麼辦了……！<hex:02100103>請你一定要對他說出「seek the sun」，把他挽留下來。<hex:02100103>他應該是前往了抱擁天坑……`
`56,TEXT_KINGZD106_05105_SYSTEM_000_008,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入含有「seek the sun」的詞句呼喚霍比格族男性。`
`58,TEXT_KINGZD106_05105_SYSTEM_000_010,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入含有「seek the sun」的詞句呼喚霍比格族男性。`
`59,TEXT_KINGZD106_05105_SAYTODO_000_011,"seek the sun"`

`25,TEXT_KINGZB108_05048_TODO_01,在「說話」頻道下輸入「Gorrl Karryor」<hex:02100103>呼喚焦躁的其瓦固一族`
`52,TEXT_KINGZB108_05048_ANXIOUSYOKFUY05048_000_006,如果還是不行的話，就請對他們說「Gorrl Karryor」。<hex:02100103>這是自古流傳的尤卡巨人族的一句話……<hex:02100103>相信他們也一定會對這句話產生反應的。`
`53,TEXT_KINGZB108_05048_ANXIOUSYOKFUY05048_000_0010,如果我的朋友和鹽田的人產生了糾紛，就先「安撫」對方，<hex:02100103>如果還是不行的話，就請對他們說「Gorrl Karryor」吧！`
`60,TEXT_KINGZB108_05048_SYSTEM_000_025,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入含有「Gorrl Karryor」的詞句。`
`63,TEXT_KINGZB108_05048_SYSTEM_000_031,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入含有「Gorrl Karryor」的詞句。`
`75,TEXT_KINGZB108_05048_SAYTODO_000_100,"Gorrl Karryor"`

`1,TEXT_KINGMA111_04870_SEQ_01,在向各位村民打招呼的同時宣布今年將正常舉辦展翅節。將聊天頻道設定為「說話」頻道後，使用鍵盤或軟鍵盤輸入含有「Ohokaliy」的詞句，向克扎瑪烏卡濕地哈努聚落的哈努族打招呼。`
`24,TEXT_KINGMA111_04870_TODO_00,在「說話」頻道下輸入「Ohokaliy」<hex:02100103>向哈努聚落的哈努族打招呼`
`57,TEXT_KINGMA111_04870_WUKLAMAT_000_033,我明白，不就是Ohokaliy嘛！`
`58,TEXT_KINGMA111_04870_SYSTEM_000_034,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入包含「Ohokaliy」的詞句，<hex:02100103>向哈努聚落的哈努族宣布展翅節即將照常舉辦。`
`61,TEXT_KINGMA111_04870_SAYTODO_000_046,Ohokaliy`
`63,TEXT_KINGMA111_04870_SYSTEM_000_051,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入包含「Ohokaliy」的詞句，<hex:02100103>向哈努聚落的哈努族宣布展翅節即將照常舉辦。`

`69,TEXT_KINGMA111_04870_SYSTEM_000_057,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入包含「Ohokaliy」的詞句，<hex:02100103>向哈努聚落的哈努族宣布展翅節即將照常舉辦。`
`73,TEXT_KINGMA111_04870_SYSTEM_000_061,將聊天頻道設定為「說話」頻道後，<hex:02100103>使用鍵盤或軟鍵盤輸入包含「Ohokaliy」的詞句，<hex:02100103>向哈努聚落的哈努族宣布展翅節即將正常舉辦。`
-----------------------------
Specialshop
1770704,星戰士裝備交易（660品級·戰鬥菁英）
1770705,星戰士裝備交易（660品級·魔法導師） 
(?660 這可能要實際查一下 )
1770786,紫票交易（100級以上釣餌·素材·票據相關）
1770783,紫票交易（100級以上素材·房屋相關）
(是橙票)
-------------------
Quest
70686, 今日的蔬菜選單
(菜單不是選單)
----------------
LeveClient
31,巴斯卡隆酒家老闆：巴斯卡隆
(去掉: 整份文件要統一規則)
-----------
leve
1767,製作委託：名產恩惠之灣的番茄水煮魚,雖然選單上
(菜單)
213,製作委託：面向騎兵的頭盔,我們想往庫爾札斯地區的宗教城市
(城邦)
--------------------
JournalGenre  請更改成
138,支線任務：活著的記憶
144,雙劍師任務
178,蝰蛇劍士任務
184,大地資源部任務
----------------
item
游擊 ?遊擊 ? 
游牧? 遊牧?
2024年度還沒結束吧?

`39918,陳舊的特殊地圖4(S4),,裝在瓶子裡的陳舊地圖。<hex:02100103>可以通過「解讀」來破譯上面的具體內容。<hex:02100103>推薦等級：90級8人小隊<hex:02100103>必定會出現傳送魔紋,陳舊的特殊地圖4(S4)`
`40516,2023年度水晶衝突區域錦標賽冠軍之證,,以獎勵在2023年度水晶衝突區域錦標賽中取得優異成績而頒發的證書。<hex:02100103>[使用後可以獲得成就],2023年度水晶衝突區域錦標賽冠軍之證`
`40519,2023年度水晶衝突區域錦標賽晉級決賽之證,,以獎勵在2023年度水晶衝突區域錦標賽中取得優異成績而頒發的證書。<hex:02100103>[使用後可以獲得成就],2023年度水晶衝突區域錦標賽晉級決賽之證`
`40520,2023年度水晶衝突區域錦標賽突破預選之證,,以獎勵在2023年度水晶衝突區域錦標賽中取得優異成績而頒發的證書。<hex:02100103>[使用後可以獲得成就],2023年度水晶衝突區域錦標賽突破預選之證`
`42353,嵌齒象革詠咒襯褲,,,嵌齒象革詠咒襯褲`
`43556,陳舊的銀狼革地圖（G16）,,裝在瓶子裡的陳舊地圖。<hex:02100103>可以通過「解讀」來破譯上面的具體內容。<hex:02100103>推薦等級：100級,陳舊的銀狼革地圖（G16）`
`43557,陳舊的獰豹革地圖（G17）,,裝在瓶子裡的陳舊地圖。<hex:02100103>可以通過「解讀」來破譯上面的具體內容。<hex:02100103>推薦狀態：100級8人小隊,陳舊的獰豹革地圖（G17）`
43986,白胡椒
44085,紙杯甜鬆糕
-----------
HWDInfoboardArticletransient
`45,「庫啵好運道」是與交納物資聯動的重建抽獎活動。<hex:02100103><hex:02100103>將庫啵好運道指定的委託品交納給波特金後，將會獲得印章。<hex:02100103>攢齊5個印章後就能進行1次抽獎。<hex:02100103><hex:02100103>抽獎卡分為左側1格，右側3格，可以選擇刮開其中一格。<hex:02100103>左側格子中可以刮出2等到4等獎，<hex:02100103>右側格子中可以刮出1等到5等獎。<hex:02100103><hex:02100103>獎品的內容每次都會發生變化，<hex:02100103>詳細內容請確認獎品表。<hex:02100103>說不定還有只能通過抽獎才可以獲得的獎品哦？！<hex:02100103><hex:02100103>對此有興趣的人，請找庫啵好運道窗口的莉茲貝絲了解詳情！,`
(說不定還有隻能<=>說不定還有只能)`

**Comment by a61456711** [comment]
真是猝不及防的釋出！＼（〇_ｏ）／
感謝[GpointChen](https://github.com/GpointChen)作者的提供，能小額斗內就好了(❁´◡`❁)
